### DISCUSSION TITLE
Installing LeApp within Linux/Kali linux

### DISCUSSION THREAD

**Discussion by doncalli** [Noovolari/leapp#127] [Q&A] [answered]
Hello,

Can someone pls help me with instructions on how to install LeApp within a linux distro (Kali-Linux)?
Update: what I am asking for help on, is any document or procedure that show how to install Leapp in Linux (Kali linux distro if possible)
Thanks.

**Comment by pusi77** [comment, marked as answer]
Hi doncalli, you can easily run Leapp on every Linux distro using the latest AppImage package you can find in [releases page](https://github.com/Noovolari/leapp/releases). You just need to:
- download the package
- open a terminal in the download directory
- make the package executable using _chmod +x_ (eg: chmod +x Leapp-0.5.3.AppImage)
- and finally run the package (eg: ./Leapp-0.5.3.AppImage)

Also [here](https://docs.appimage.org/introduction/quickstart.html) is a tutorial from AppImage documentation.

**Reply by pethron** [reply]
We'll provide more clear documentation.
